packages/angular/projects/ui-angular/src/lib/components/authenticator/components/sign-up/sign-up-form-fields/sign-up-form-fields.component.html

```diff
@@ -11,15 +11,25 @@
         [dialCode]="userOverrides?.dialCode"
         [dialCodeList]="userOverrides?.dialCodeList"
       ></amplify-user-name-alias>
-      <amplify-form-field
-        *ngSwitchCase="'password'"
-        [labelHidden]="grabField('password', 'labelHidden', true)"
-        [placeholder]="grabField('password', 'placeholder', null)"
-        [required]="grabField('password', 'required', true)"
-        [label]="grabField('password', 'label', null)"
-        name="password"
-        autocomplete="new-password"
-      ></amplify-form-field>
+      <ng-container *ngSwitchCase="'password'">
+        <amplify-form-field
+          [labelHidden]="grabField('password', 'labelHidden', true)"
+          [placeholder]="grabField('password', 'placeholder', null)"
+          [required]="grabField('password', 'required', true)"
+          [label]="grabField('password', 'label', null)"
+          name="password"
+          autocomplete="new-password"
+        ></amplify-form-field>
+        <div data-amplify-sign-up-errors *ngIf="errors">
+          <div
+            class="amplify-text"
+            data-variation="error"
+            *ngFor="let error of errors"
+          >
+            {{ translate(error) }}
+          </div>
```

Comment:
I could have used the Angular error component, but it didn't look right here and didn't match react and Vue. So I used a div instead.

Reply:
Ah gotcha, I'll add a TODO item to handle it later then.

Reply:
👍